cinema/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.CharField(max_length=255)
+    duration = models.IntegerField()
+
+    def __str__(self):
+        return str(self.title)
```

Comment:
title is already str u don't need to use str() for concertation
add typehint

Reply:
misunderstood your last comment